rskj-core/src/main/java/co/rsk/rpc/modules/eth/EthModule.java

```diff
@@ -175,6 +175,12 @@ public String estimateGas(CallArguments args) {
         }
     }
 
+    public String estimateGas(CallArguments args) {
+        Block bestBlock = blockchain.getBestBlock();
+        return estimateGas(args, bestBlock);
+    }
```

Comment:
not in use?

Reply:
It is being used in test, but I didn't removed it in case it could be used as a lib by powpeg or others and don't break compatibility

Reply:
seems it's not used in powpeg. also the int. tests will would fail